site/content/docs/5.2/components/list-group.md

```diff
@@ -144,6 +149,11 @@ Add `.list-group-horizontal` to change the layout of list group items from verti
 
 **ProTip:** Want equal-width list group items when horizontal? Add `.flex-fill` to each list group item.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications.
```

Comment:
For example like that, we should say something like "This is just an example showing the layout utility, please don't use it as is in your website blah blah blah".
TBH the generic approach makes me think "what doesn't respect the ODS guidelines", it's just a layout.

Reply:
Resolved ?